.travis.yml

```diff
@@ -1,11 +1,11 @@
 sudo: required
 language: generic
-dist: trusty
+dist: xenial
```

Comment:
I just saw this and changed it- are you really running Trusty (14.04)? Kinetic should be run on 16.04, not that it matters it's all running in Docker.